packages/ds-healthcare-gov/src/components/Accordion/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { AccordionItem } from '@cmsgov/design-system';
+import { MinusCircleIcon, PlusCircleIcon } from '@cmsgov/design-system';
+
+AccordionItem.defaultProps = {
+  ...AccordionItem.defaultProps,
+  // closeIcon: (
+  //   <MinusCircleIcon
+  //     className="ds-c-accordion__button-icon"
+  //     title={t('accordion.close')}
+  //     ariaHidden={false}
+  //   />
+  // ),
+  // openIcon: (
+  //   <PlusCircleIcon
+  //     className="ds-c-accordion__button-icon"
+  //     title={t('accordion.open')}
+  //     ariaHidden={false}
+  //   />
+  // ),
+};
```

Comment:
Getting errors when overwriting the default 'closeIcon' & 'openIcon' on hcgov. I'm not sure why & still debugging this.
![Screen Shot 2022-06-22 at 3 46 56 PM](https://user-images.githubusercontent.com/20322880/175123333-fb3c6195-358d-4d01-926c-82c8cd8f55e6.png)

error:

Reply:
I pulled this branch locally, ran `yarn build` to build the types for the core repo. Then I ran `yarn build:healthcare` to re-build the healthcare DS to use the new types from core.

Then, I had to add 
```
import React from 'react';
```
to the top of this file in order to use JSX elements and the error went away for me. Ni, can you try these steps and let me know how they work out?